vignettes/spatial_wwinference.Rmd

```diff
@@ -0,0 +1,2457 @@
+  guides(
+    fill = guide_legend(
+      title = "Assumed Corr. Structure"
+    ),
+    color = guide_legend(
+      title = "Assumed Corr. Structure"
+    )
+  ) +
+  scale_fill_manual(
+    breaks = c("Exponential", "Unstructured", "IID"),
+    values = c("darkviolet", "deeppink3", "darksalmon")
+  ) +
+  scale_colour_manual(
+    breaks = c("Exponential", "Unstructured", "IID"),
+    values = c("darkviolet", "deeppink3", "darksalmon")
+  ) +
+  theme_bw()
+# ------------------------------------------------------------------------------
+# Wastewater results -----------------------------------------------------------
+ww_ribbon_data <- all_pred_draws_df %>%
+  filter(
+    name == "predicted wastewater"
+  ) %>%
+  mutate(
+    site_lab_name = glue::glue("{subpop}, Lab: {lab}")
+  ) %>%
+  group_by(
+    date,
+    subpop,
+    inf_model_type,
+    gen_model_type
+  ) %>%
+  summarise(
+    lower = quantile(pred_value, 0.025, na.rm = TRUE),
+    median = median(pred_value),
+    upper = quantile(pred_value, 0.975, na.rm = TRUE),
+    .groups = "drop"
+  )
+ww_result_plot <- ggplot(
+  all_pred_draws_df %>%
+    filter(
+      name == "predicted wastewater"
+    )
+) +
+  geom_ribbon(
+    data = ww_ribbon_data,
+    aes(x = date, ymin = lower, ymax = upper, fill = inf_model_type),
+    alpha = 0.1
+  ) +
+  geom_line(
+    data = ww_ribbon_data,
+    aes(x = date, y = median, color = inf_model_type),
+    size = 1,
+  ) +
+  geom_point(
+    aes(x = date, y = observed_value)
+  ) +
+  geom_vline(aes(xintercept = lubridate::ymd(forecast_date)),
+    linetype = "dashed"
+  ) +
+  xlab("") +
+  ylab("Genome copies/mL on Log Scale") +
+  facet_grid(subpop ~ gen_model_type, scales = "free_y") +
+  guides(
+    fill = guide_legend(
+      title = "Assumed Corr. Structure"
+    ),
+    color = guide_legend(
+      title = "Assumed Corr. Structure"
+    )
+  ) +
+  scale_fill_manual(
+    breaks = c("Exponential", "Unstructured", "IID"),
+    values = c("darkviolet", "deeppink3", "darksalmon")
+  ) +
+  scale_colour_manual(
+    breaks = c("Exponential", "Unstructured", "IID"),
+    values = c("darkviolet", "deeppink3", "darksalmon")
+  ) +
+  theme_bw()
+# ------------------------------------------------------------------------------
+
+(hosp_result_plot / ww_result_plot) +
+  plot_layout(guides = "collect") &
+  theme(legend.position = "bottom")
+```
+
+## Rt curves, global and site
+Here we showcase the global and site level Rt's.  Notation is the same as
+hospital admissions and wastewater results, except here the actual global and
+site Rt curves are plotted for visual evaluation.  These actual curves are not
+used by the model, so these curves must be inferred by the model.
+
+We do very well to capture the actual Rt curves, there is some concern with how
+high the inferred Rt reaches toward the nowcasting and forecasting periods.  This
+is not surprising as there is no data there to guide the model for a more
+accurate quantile.
```

Comment:
```suggestion
high the inferred Rt reaches toward the nowcasting and forecasting periods.
We expect to see higher uncertainty in the nowcasting and forecasting periods
because the Rt is only partially or not at all informed by recent data. 
```